### PR TITLE
Run compilemessages django command at Dockerfile, not at startup script

### LINKED ISSUE #910: Do not run `compilemessages` in `gunicorn.sh`
Why?
1. It will create a file that can be dropped by fs, since it is not in the image
2. We can run celery or something without direct `gunicorn.sh` call

Let's move it to the image creation stage.

### PR DESCRIPTION
**Pull request by AlwxSin**
Run command in Dockerfile after all code is copied into image.

Closes #910

### CODE REVIEW

File: {{cookiecutter.project_name}}/docker/django/Dockerfile [hunk 1]
```diff
@@ -71,3 +71,5 @@ ENTRYPOINT ["/sbin/tini", "--", "/docker-entrypoint.sh"]
 FROM development_build as production_build
 
 COPY . /code
+
+RUN python /code/manage.py compilemessages
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/docker/django/Dockerfile]: This way we won't have compiled messages in development. What do you think?

Comment (sobolevn) [reply {{cookiecutter.project_name}}/docker/django/Dockerfile]: We currently don't have them as well

Comment (AlwxSin) [reply {{cookiecutter.project_name}}/docker/django/Dockerfile]: Why we should have them in development?

Besides, in development, we mount code, not copy it. If a developer needs something to do with messages, he will mount them and compile manually, isn't it?

Comment (sobolevn) [reply {{cookiecutter.project_name}}/docker/django/Dockerfile]: Good enough!

**Pull request merged**